core/src/main/scala/chisel3/internal/firrtl/Converter.scala

```diff
@@ -427,11 +431,16 @@ private[chisel3] object Converter {
     case t: Property[_] => t.getPropertyType
   }
 
-  def convert(name: String, param: Param): fir.Param = param match {
+  def convert(name: String, param: Param, idToComponent: VectorMap[BaseModule, Component]): fir.Param = param match {
     case IntParam(value)    => fir.IntParam(name, value)
     case DoubleParam(value) => fir.DoubleParam(name, value)
     case StringParam(value) => fir.StringParam(name, fir.StringLit(value))
-    case RawParam(value)    => fir.RawStringParam(name, value)
+    case PrintableParam(value, id) => {
+      val ctx = idToComponent.get(id).get
```

Comment:
Style nit that's irrelevant because we should get rid of the map, but `myMap.get(key).get` is the same thing as just `myMap(key)`, but see below:

`_component` is a package private field of `BaseModule`: https://github.com/chipsalliance/chisel/blob/9448d32685b102f23bd6d97923fc45294635e1d0/core/src/main/scala/chisel3/Module.scala#L829

You shouldn't need this map, just use `id._component.get`.  It's an `Option` because it is set at the end of Module construction, but that means it will be set before Conversion so it's safe to `.get`.